### DISCUSSION TITLE
"git" failed to start: "No such file or directory" when running checkout on Docker

### DISCUSSION THREAD

**Discussion by npapagna** [theory/sqitch#823] [Q&A]
I believe this is because the `git` command is not available in the Docker image.

Is there a recommended way to make it work?

Thanks in advance.

**Comment by theory** [comment]
I presume you're trying to use `git` in the ]Sqitch docker image](/sqitchers/docker-sqitch)? You can either install it with `apt-get update && apt-get install -y git` or build your own Docker image on top of the Sqitch image.

**Reply by npapagna** [reply]
That's right. I'm trying to run `sqitch checkout` from that Docker image.
Is there a reason the image does not come with `git` pre-installed?

I don't think I can `apt-get install` into the existing Sqitch image (I'll have to build my own).
